Classes/YLTableViewDataSource.h

```diff
@@ -16,8 +16,14 @@ NS_ASSUME_NONNULL_BEGIN
 //! If one of model's properties has changed (but it's still the same object), this will reload the visible cell for that model. Useful whenever something mutates the state of a model (i.e. image loading)
 - (void)reloadVisibleCellForModel:(id)model inTableView:(UITableView *)tableView;
 
+//! If the cells implement the YLTableViewCellEstimatedRowHeight protocol, that height will be used. Otherwise, the tableView can use an alternate height specified in this method.
+- (CGFloat)estimatedHeightForRow;
```

Comment:
We should continue the pattern of passing through the table view in question. Also, we should given the client some information about which row we're asking about: ```- (CGFloat)estimatedHeightForRow:(NSIndexPath *)row inTableVIew(UITableView *)table```